docs/configuration.v2alpha1.md

```diff
@@ -374,3 +384,22 @@ In the table, `spec.override.nodeAgent.image.name` and `spec.override.nodeAgent.
 [3]: https://github.com/DataDog/datadog-operator/blob/main/examples/datadogagent/v2alpha1/datadog-agent-with-apm-hostport.yaml
 [4]: https://github.com/DataDog/datadog-operator/blob/main/examples/datadogagent/v2alpha1/datadog-agent-with-clusteragent.yaml
 [5]: https://github.com/DataDog/datadog-operator/blob/main/examples/datadogagent/v2alpha1/datadog-agent-with-tolerations.yaml
+[7]: https://github.com/DataDog/datadog-operator/blob/main/docs/configuration.v1alpha1.md
```

Comment:
This should be added to footer template here
https://github.com/DataDog/datadog-operator/blob/main/hack/generate-docs/v2alpha1_footer.markdown